src/test/1.3/scala/com/holdenkarau/spark/testing/SampleStreamingActionTest.scala

```diff
@@ -37,7 +32,7 @@ class SampleStreamingActionTest extends StreamingActionBase {
 
   def countWordsLength(acc: Accumulator[Int]): (DStream[String] => Unit) = {
     def c(input: DStream[String]): Unit = {
-      input.foreach{r: RDD[String] =>
+      input.foreachRDD{r: RDD[String] =>
```

Comment:
I think (so we can cross-compile all the way back to 1.3) we need to keep the old foreach here :(